pom.xml

```diff
@@ -112,8 +112,8 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-      <maven.compiler.source>1.7</maven.compiler.source>
-      <maven.compiler.target>1.7</maven.compiler.target>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
From a *very* practical point of view, it would be a win / win thing if only the server side inherit the 1.8 part, and client-side projects and related modules specifies the 1.7 version, overriding and pom.xml parent's definition.

Sounds good? :)